src/plugins/ui_feature/insert_pivot.ts

```diff
@@ -54,4 +56,26 @@ export class InsertPivotPlugin extends UIPlugin {
       content: `=PIVOT(${formulaId})`,
     });
   }
+
+  duplicatePivotInNewSheet(pivotId: UID, newPivotId: UID, newSheetId: UID) {
+    this.dispatch("DUPLICATE_PIVOT", {
+      pivotId,
+      newPivotId,
+    });
+    const activeSheetId = this.getters.getActiveSheetId();
+    const position = this.getters.getSheetIds().indexOf(activeSheetId) + 1;
+    const formulaId = this.getters.getPivotFormulaId(newPivotId);
+    this.dispatch("CREATE_SHEET", {
+      sheetId: newSheetId,
+      name: _t("Pivot #%s", formulaId),
```

Comment:
The name could already be used, and so the command would be rejected and the code will crash later

Reply:
it's the same issue here, no ?
https://github.com/odoo/o-spreadsheet/blob/76448b329247b270318a4b2db7ddd5daff0e933e/src/plugins/ui_feature/insert_pivot.ts#L43C1-L44C1

Reply:
Indeed. It should be fixed